<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Selection</class>
 <widget class="QDialog" name="Selection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="trailer">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>А5431В-7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>А0784Е-7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>А7529В-7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>А7320Е-7</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Место выезда</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата приезда с</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>729</x>
     <y>103</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тягач</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Driver">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сергей Котович</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Максим Кучинский</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Виктор Маркович</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Григорий Василевский</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата выезда с</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Место приезда</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Водитель</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Truck">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>АО 7120-7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>АО 6222-7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>АВ 2071-7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>АТ 6804-7</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="dateArriveFrom">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="payment">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Оплачено</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Оплачено частично</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Не оплачено</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Оплата</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateExitFrom">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Расходы</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Прицеп</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="exit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="arrival">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>40</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>№</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="no">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateExitBefore">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата выезда по</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата приезда по</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateArriveBefore">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="dataExitAgg">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>16</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="dataArriveAgg">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>16</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Driver</zorder>
  <zorder>trailer</zorder>
  <zorder>label_4</zorder>
  <zorder>label_7</zorder>
  <zorder>pushButton</zorder>
  <zorder>label</zorder>
  <zorder>label_6</zorder>
  <zorder>label_5</zorder>
  <zorder>label_3</zorder>
  <zorder>Truck</zorder>
  <zorder>dateArriveFrom</zorder>
  <zorder>payment</zorder>
  <zorder>label_8</zorder>
  <zorder>dateExitFrom</zorder>
  <zorder>label_9</zorder>
  <zorder>label_2</zorder>
  <zorder>exit</zorder>
  <zorder>arrival</zorder>
  <zorder>label_10</zorder>
  <zorder>no</zorder>
  <zorder>dateExitBefore</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>dateArriveBefore</zorder>
  <zorder>dataExitAgg</zorder>
  <zorder>dataArriveAgg</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
